pkg/invoice/client.go

```diff
@@ -61,9 +61,9 @@ func (c *client) Search(ctx context.Context, request SearchRequest) (*SearchResp
 	queryParameters := request.GetParams()
 
 	requestData := httpclient.RequestData{
-		QueryParams: queryParameters,
 		Method:      http.MethodGet,
 		URL:         urlSearch,
+		QueryParams: queryParameters,
```

Comment:
Padronizar: na maioria dos outros lugares está como queryParams. 
```suggestion
		QueryParams: queryParams,
```

Reply:
boa, corrigido!